ray-operator/controllers/ray/common/ingress.go

```diff
@@ -87,7 +86,7 @@ func BuildIngressForHeadService(cluster rayv1.RayCluster) (*networkingv1.Ingress
 	// Get ingress class name from rayCluster annotations. this is a required field to use ingress.
 	ingressClassName, ok := cluster.Annotations[IngressClassAnnotationKey]
 	if !ok {
-		logrus.Warn(fmt.Sprintf("ingress class annotation is not set for cluster %s/%s", cluster.Namespace, cluster.Name))
+		log.Info(fmt.Sprintf("ingress class annotation is not set for cluster %s/%s", cluster.Namespace, cluster.Name))
```

Comment:
It'd be preferable to avoid string formatting and use the structured form, e.g.:

```go
log.Info("ingress class annotation is not set for cluster", cluster.Namespace, cluster.Name)
``` 

Maybe you're already planning to do it in a separate PR.

Reply:
Well, I am planning to add the namespace to the structured form, but I am not sure whether should we move all the logging parameters to the structured form. IMHO, the structured fields are subject to special purposes only such as filtering. Moving all parameters out from the message may reduce readability.

Reply:
Right, having both the formatted message and the structured info sounds reasonable too.